.github/workflows/ci.yml

```diff
@@ -36,18 +36,42 @@ jobs:
               - package.json
               - package-lock.json
               - src/app/**
-
   install:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: 14
+          node-version: 18
+          cache: 'npm'
+          cache-dependency-path: '**/package-lock.json'
+      - run: npm ci --legacy-peer-deps
+
+  build:
+    needs: install
+    runs-on: ubuntu-latest
+    env:
+      NODE_OPTIONS: '--max-old-space-size=4096 --openssl-legacy-provider'
+    steps:
+      - uses: actions/checkout@v2
+      - run: mkdir -p build-output/
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
-      - run: npm ci
+      - run: npm ci --legacy-peer-deps
+      - run: npm run build
+      - name: Upload build files
+        uses: actions/upload-artifact@v2
+        if: always()
```

Comment:
what does that mean? 😅 . Is the whole if conditional unnecessary if it always runs?